neptune/alpha/internal/utils/sync_offset_file.py

```diff
@@ -19,10 +19,12 @@
 
 
 class SyncOffsetFile:
-    def __init__(self, experiment_path: Path):
-        offset_file_path = experiment_path / 'offset'
-        mode = 'r+' if offset_file_path.exists() else 'w+'
-        self._file = open(offset_file_path, mode)
+    def __init__(self, filename: str, default: int = None):
```

Comment:
I would try to use pathlib.Path throughout the codebase instead of string paths.

Reply:
Ok.